includes/Heartbeat.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook;
```

Comment:
Should this be in `Utilities` or some other folder? Probably a good idea to avoid adding stuff to `includes` root.

Reply:
Sounds good, moved in 65fd9e2a6c74743c78478c1149f1cc80ddc0e5d4